ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.create-run.tsx

```diff
@@ -421,6 +485,135 @@ const CreateRunPage = () => {
               )}
             />
 
+            <h3 className='mt-4'>Parameters</h3>
+            <div className='text-sm text-gray-500'>
+              A map with custom parameters for the whole ORT run. The parameters
+              can be evaluated by the validation script executed by the config
+              worker. The script can convert these parameters to specific job
+              configurations understood by ORT Server. The first column is the
+              key and the second is the value.
```

Comment:
Regarding this last sentence, can we instead have "Key" and "Value" headers for the columns?